app/plugins/ldap-authentication-advanced.md

```diff
@@ -0,0 +1,23 @@
+---
+id: page-plugin
+title: Plugins - LDAP Authentication Advanced
+layout: plugin-ee
+header_title: LDAP Authentication Advanced
+header_icon: https://konghq.com/wp-content/uploads/2018/07/EE-ldap.png
+breadcrumbs:
+  Plugins: /plugins
+description: |
+  The LDAP Authentication Advanced plugin for Kong Enterprise Edition is an extended version of the Community Edition LDAP Authentication plugin, with enhanced configuration options and features.
+
+---
+
+## Enterprise Features
+
+#### Consumer Mapping
+
+When authenticating via Kong, it is sometimes advantageous to apply plugin behavior at the consumer level (e.g. rate-limiting)
```

Comment:
ditto

Reply:
Maybe "When authenticating via Kong, it is sometimes advantageous to apply plugin behavior at the consumer level (e.g., rate-limiting); as such, the following options are provided:"?